apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -384,13 +384,13 @@ static void fs_vfs_dup_tc(void)
 
 	/* Nagative case with invalid argument, invalid fd. It will return ERROR */
 #if CONFIG_NFILE_DESCRIPTORS > 0
-	ret = dup(CONFIG_NFILE_DESCRIPTORS);
-	TC_ASSERT_EQ("dup", ret, ERROR);
+	fd1 = dup(CONFIG_NFILE_DESCRIPTORS);
+	TC_ASSERT_LT_CLEANUP("dup", fd1, 0, close(fd1));
```

Comment:
The "fd1 < 0" means failure of dup function. Why do you use cleanup?

Reply:
That means "fd1>=0" because a condition of TC_ASSERT_LT_CLEANUP macro is "var >= ref".
So close(fd1) is needed in that case.